code/modules/mob/living/living.dm

```diff
@@ -275,6 +275,10 @@ Sorry Giacom. Please don't be mad :(
 /mob/living/proc/setMaxHealth(var/newMaxHealth)
 	maxHealth = newMaxHealth
 
+/mob/living/proc/getTotalLoss(var/amount)
+	amount = getBruteLoss() + getOxyLoss() + getToxLoss() + getFireLoss() + getCloneLoss() + getBrainLoss()
+	return amount
+

```

Comment:
Why are you adding this proc, it's unused as far as I can see. Also it makes no sense to include getBrainLoss here.